### ISSUE TITLE
Spike: Take part at the AXE DeveloperHub (Watcher) beta program and integrate their new service

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2026]
Scope
---
Deque is developing a new product named **AXE DeveloperHub** which is currently in open beta.

AXE DeveloperHub is a tool to setup automated accessibility testing and is directly connected to GitHub. It tracks, monitors and compares all reported issues automatically branch- and commit based. Therefor it could help developers to have a detailed overview of all issues in their working environment.

We have the opportunity to get an early access to the beta program to test out the tool. This includes an onboarding session and access to their services.

**Outcome:**
- Understand the tool and all of its functionalities and possibilities
- Evaluate the tool and its pros and cons
- Give feedback to Deque to help them in further development

**Contact:** Deque product manager: [keith.rhodes@deque.com](mailto:keith.rhodes@deque.com) | [dylan.barrell@deque.com](mailto:dylan.barrell@deque.com)

Out of Scope
---
No further accessibility tests should be created. Only use exisitng tests from Storefront.

Notes  
---
- We should use our existing testing results stored in `.json` format and find a way how to pass them with every commit to their service.
- Get Started: https://axe.deque.com/axe-watcher/startup 
- Documentation: https://docs.deque.com/developer-hub/2/en/dh-welcome

Acceptance criteria
---
-  ~~AXE DeveloperHub is integrated into Porsche Design System repo in GitHub to evaluate the service~~
- [x] AXE DeveloperHub is tested with a predefined sample project from Deque

Subtasks
---
- [ ] Task
